changelogs/unreleased/gh-4693-formats-for-spaceless-tuples.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
+
+* Introduced `box.tuple.format` that enables format definition for tuples
+  created via `box.tuple.new`(spaceless tuples) (gh-4693).
```

Comment:
You didn't mention here the breaking change about `box.tuple.new(...)` not working anymore unless a compat option is changed to `'old'`. There should be a "breaking change" label and the compat option mentioned. Could be a separate point in the same changelog file.

Reply:
Not sure this should be published and needs a changelog: the argument list syntax was never mentioned in the documentation (@unera and @Totktonada can confirm) was never mentioned in the documentation.

Reply:
This is a gray zone. I would add the 'breaking change' mark, but leave a note that the old behavior was never documented and unlikely used by actual applications/modules (please, re-check this point; I did, but it was some time ago and I'm not 100% sure now).

Reply:
Added a seperate changelog.